File: app/src/main/java/com/ddpc/ggway/ui/activity/user/update/PresenterUpdateUser.java
package com.ddpc.ggway.ui.activity.user.update;

import com.ddpc.ggway.ui.MainPresenter;

/**
 * Created by diha- on 03.02.2018.
 */

public interface PresenterUpdateUser extends MainPresenter {
    void onUpdateButtonClick(String userId);
    void onItemClicked(int position);
}

File: app/src/main/java/com/ddpc/ggway/ui/activity/user/update/PresenterUpdateUser.kt
package com.ddpc.ggway.ui.activity.user.update

import com.ddpc.ggway.ui.MainPresenter

/**
 * Created by diha- on 03.02.2018.
 */

interface PresenterUpdateUser : MainPresenter {
    fun onUpdateButtonClick(userId: String)
    fun onItemClicked(position: Int)
}
